plugins/generator-1.19.2/forge-1.19.2/templates/gui/gui_container.java.ftl

```diff
@@ -115,6 +115,7 @@ public class ${name}Menu extends AbstractContainerMenu implements Supplier<Map<I
 
 						<#if hasProcedure(component.disablePickup) || component.disablePickup.getFixedValue()>
 						@Override public boolean mayPickup(Player entity) {
+							int slot = ${component.id};

```

Comment:
This could probably be field of inner-class to not have to define it in each method

Reply:
✔️ Fixed